packages/webpack/src/config/server.js

```diff
@@ -20,10 +20,13 @@ export default class WebpackServerConfig extends WebpackBaseConfig {
     const whitelist = [
       /\.(?!js(x|on)?$)/i
     ]
-    for (const pattern of this.buildContext.buildOptions.transpile) {
+    for (let pattern of this.buildContext.buildOptions.transpile) {
```

Comment:
Looks like we have a duplicate code block here (as in `src/config/base.js`, L61-70). Could we extract it somehow?

Reply:
Could be, but please note the difference in `path.normalize`

Reply:
I will do it in another pr since this has been a long time